src/components/ui/Icon.vue

```diff
@@ -450,15 +450,9 @@
       </g>
     </template>
     <template v-else-if="id === 'zoom'">
-      <polygon fill="currentColor" points="23 7 16 12 23 17 23 7" />
-      <rect
-        fill="currentColor"
-        x="1"
-        y="5"
-        width="15"
-        height="14"
-        rx="2"
-        ry="2"
+      <path
+        id="a"
```

Comment:
Not sure this `id` is needed, we should probably also add `fill="currentColor"` like the other svgs in this file.

Reply:
Nice. that's been fixed.